tools/sense_studio/templates/train.html

```diff
@@ -0,0 +1,63 @@
+{% extends 'skeleton.html' %}
+
+{% block title %} {{ project }} {% endblock %}
+
+{% block navigation %}
+    {% include 'navigation.html' %}
+{% endblock navigation %}
+
+{% block main %}
+<div id="main" class="ui main text container">
+    <h1 class="ui header">
+        Train Model
+    </h1>
+    <form class="ui form" action="{{ url_for('.train_model') }}" method="post">
+        <input type="hidden" name="project" value="{{ project }}">
+        <div class="two fields wide">
+            <div class="field">
+                <label>Choose Model</label>
+                <div class="ui selection dropdown train">
+                    <input type="hidden" name="model_name" value="{{ models[0] }}">
+                    {% if models %}
+                        <div class="default text">Select Model</div>
+                        <i class="dropdown icon"></i>
+                        <div class="menu">
+                            {% for model in models %}
+                                <div class="item" data-value="{{ model }}">{{ model }}</div>
+                            {% endfor %}
+                        </div>
+                    {% else %}
+                       <div class="default text">No models to select</div>
+                    {% endif %}
+                    </div>
+                </div>
+            <div class="field train">
+                <label>Number of layers to finetune</label>
+                <input type="number" name="layers_to_finetune" min="5" value="5">
+            </div>
+        </div>
+        <div class="two fields wide">
+            <div class="field train">
+                <label>Epochs</label>
+                <input type="number" name="epochs" min="80" value="80">
+            </div>
+            <div class="field train">
+                <label>Output folder</label>
+                <input type="text" name="output_folder" placeholder="Folder Name">
+            </div>
+        </div>
+      <button class="ui button" id="trainModel" type="submit" tabindex="0" {% if is_disabled %} disabled="disabled" {% endif %}>Train model</button>
+      <button class="ui button" type="submit" tabindex="1" formaction="{{ url_for('.cancel_training') }}">Cancel</button>
+    </form>
+    <div class="ui segment train-console">
+        {% if is_disabled %}
+            <h4 class="font-style">Training Model...</h4>
+        {% endif %}
+        <h4 class="font-style">
+            {% for log in logs %}
+                {{ log }} <br />
+            {% endfor %}
```

Comment:
Is it possible to make this area auto-scroll to the most recent line?

Reply:
Finding a way on that already have in mind